<launch>

<!-- ################################# -->
<!-- inch manipulator -->
  <arg name="robot_name"			default="/inch"/>
  <arg name="length_1"				default="0.24215"/>
  <arg name="length_2"				default="0.1759"/>
  <arg name="length_3"				default="0."/>

  <node name="inch_manipulator_control" pkg="inch_controllers" type="inch_manipulator_control" required="true" output="screen"> 
  </node>

</launch>
